.github/workflows/clojure.yml

```diff
@@ -0,0 +1,21 @@
+name: Clojure CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '1.8' # The JDK version to make available on the path.
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64 # (x64 or x86) - defaults to x64
+    - uses: DeLaGuardo/setup-clojure@2.0
+      with:
+        tools-deps: '1.10.1.469'
+    - name: Install dependencies
+      run: sudo apt-get update; sudo apt-get install rlwrap 
```

Comment:
Do we need rlwrap for non-interactive use?

Reply:
dont need it so i've cleaned up the setup file